events/wc_undead_events.txt

```diff
@@ -1457,6 +1626,67 @@ character_event = {
 			}
 		}
 	}
+	
+	option = {
+		trigger = {
+				religion = throsic
+		}
+		name = EVTOPTB_WCUND_355_wickergolem
```

Comment:
Don't see the point of the new option. You could just relocalize the name.
https://ck2.paradoxwikis.com/Event_modding#Option_Name